### PR TITLE
typeSource const FactStreamType

### PR DESCRIPTION
**Pull request by OlexandrNazin**

### CODE REVIEW

File: surface/model_openapiv3.go [hunk 1]
```diff
@@ -44,6 +44,14 @@ func (b *OpenAPI3Builder) buildModel(document *openapiv3.Document) (*Model, erro
 	b.model.Name = document.Info.Title
 	b.model.Types = make([]*Type, 0)
 	b.model.Methods = make([]*Method, 0)
+	for _, v := range document.Components.SpecificationExtension {
+		if v.Name == "x-command-stream-types" {
```
Comment (taaraora) [inline surface/model_openapiv3.go]: Why do we need this check?
Let's just take all extension properties.

File: surface/surface.proto [hunk 2]
```diff
@@ -68,6 +68,7 @@ message Type {
 
     string typeName = 6; // language-specific type name
     repeated NamedAny specification_extension = 7;
+    string typeSource = 100;
```
Comment (taaraora) [inline surface/surface.proto]: I think we don't need this field.

**Review by taaraora — CHANGES_REQUESTED** [review]

File: surface/model_openapiv3.go (hunk 1 above)
Comment (OlexandrNazin) [reply surface/model_openapiv3.go]: I made the necessary corrections

**Review by taaraora — APPROVED** [review]
Thank you @OlexandrNazin.

File: surface/surface.proto (hunk 2 above)
Comment (OlexandrNazin) [reply surface/surface.proto]: I agree with you, removed

**Pull request merged**